src/garrysmod/gamemodes/zombiesurvival/gamemode/howl/sv_howl.lua

```diff
@@ -0,0 +1,261 @@
+Howl = {}
+Howl.logger = include("lib/log.lua")
+Howl.commands = {}
+Howl.roles = {
+    ["admin"] = {
+        root = true,
+        -- These are skipped due to root and are for illustrative purposes only.
+        perms = { "administrate", "moderate" }
+    },
+    ["moderator"] = {
+        perms = { "moderate" }
+    },
+    ["user"] = {
+        perms = {}
+    }
+}
```

Comment:
The roles for the admin mod are static and defined here. I didn't see a reason for this to be more robust, but in the future, we could certainly have these declared in the database or something.